### ISSUE TITLE
Missing variable definition in Object to RDF Conversion

### ISSUE THREAD

**Issue by kasei** [w3c/json-ld-api#277]
[Object to RDF Conversion](https://w3c.github.io/json-ld-api/#object-to-rdf-conversion) step 13.1 says:

> If `rdfDirection` is `i18n-datatype`, set `datatype` to the result of appending `language` and the value of `@direction` in `item` separated by an underscore (`"_"`) to `https://www.w3.org/ns/i18n#`.

However, the `language` variable does not seem to be set prior this step.

**Comment by gkellogg** [comment]
It should just reverence the `@language` value in _item_, if any. Perhaps the following would solve this:

> If `rdfDirection` is `i18n-datatype`, set _datatype_ to the result of appending the value of `@language` in _item_, or the empty string (`""`) if there is no such value, with the value of `@direction` in _item` separated by an underscore (`"_"`) to ..."

**Comment by kasei** [comment]
Agreed, your proposed text would solve this. But it gets a bit wordy and complex by trying to (I assume) avoid splitting out a new temporary variable declaration for "@language in item, or the empty string ('') if there is no such value".

**Comment by gkellogg** [comment]
@kasei see update in PR #286. Please indicate if this addresses the issue.

**Comment by kasei** [comment]
@gkellogg looks good, thanks!
